packages/react-components/react-drawer/library/etc/react-drawer.api.md

```diff
@@ -172,7 +172,7 @@ export type OverlayDrawerSlots = {
 
 // @public
 export type OverlayDrawerState = ComponentState<OverlayDrawerInternalSlots> & Required<DrawerBaseState> & Pick<OverlayDrawerProps, 'mountNode'> & {
-    hasMountNodeElement?: boolean;
+    hasMountNodeElement: boolean;
```

Comment:
Why did this change? Is it breaking?

Reply:
I believe after pulling fresh master, it has to be incorrect there 😕 

Coming from recent PR from @marcosmoura: #33226  

So I guess, it was generated, then there was this fix: https://github.com/microsoft/fluentui/pull/33226/commits/2f2c79607115deb12ddfff8b6bb4c49dc013a023 

and it was automerged after, probably the change was not spotted because of the nx cache 

> Why did this change? Is it breaking?

Nope. The state is required and not optional, so it's expected 🙂